### PR TITLE
Integration test duration

### PR DESCRIPTION
**Pull request by chaodaiG**

### CODE REVIEW

File: prow/test/integration/integration-test.sh [hunk 1]
```diff
@@ -39,4 +39,6 @@ main() {
   "${bazel}" test //prow/test/integration/test:go_default_test --action_env=KUBECONFIG=${HOME}/.kube/config --test_arg=--run-integration-test "$@" || ( echo "FAILED: running tests">&2; return 1 )
 }
 
-main "$@"
+time bazel test --config=ci --nobuild_tests_only //...
```
Comment (fejta) [inline prow/test/integration/integration-test.sh:42]: Please include it with L39 so things run in parallel

Comment (chaodaiG) [reply prow/test/integration/integration-test.sh]: Test flag doesn't work, I can switch it to use env var

Comment (fejta) [reply prow/test/integration/integration-test.sh]: Another option would be to put the arg into the sh_test or whatever target

**Comment by chaodaiG** [comment]
First run succeeded with 12m20s https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/test-infra/21044/pull-test-infra-integration/1365374671578468352

Run it again
/test pull-test-infra-integration

**Comment by chaodaiG** [comment]
Second run succeeded in 18m54s https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/test-infra/21044/pull-test-infra-integration/1365383348578422784

Run it the third time
/test pull-test-infra-integration

**Comment by fejta** [comment]
Remember since this job will run on every PR the important thing to think about is the critical path time

**Comment by chaodaiG** [comment]
The third run was 12m39s
https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/test-infra/21044/pull-test-infra-integration/1365388607161896960

**Comment by chaodaiG** [comment]
12m1s https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/test-infra/21044/pull-test-infra-integration/1365397741399707648
/test pull-test-infra-integration

**Comment by chaodaiG** [comment]
10m28s https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/test-infra/21044/pull-test-infra-integration/1365401019470581760
/test pull-test-infra-integration

**Comment by chaodaiG** [comment]
10m24s https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/test-infra/21044/pull-test-infra-integration/1365405811035082752
/test pull-test-infra-integration

**Comment by chaodaiG** [comment]
13m8s https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/test-infra/21044/pull-test-infra-integration/1365409724618510336

**Comment by chaodaiG** [comment]
There are 7 data points now, I think they should be good for a comparison.

- 12m20s https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/test-infra/21044/pull-test-infra-integration/1365374671578468352
- 18m54s https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/test-infra/21044/pull-test-infra-integration/1365383348578422784
- 12m39s https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/test-infra/21044/pull-test-infra-integration/1365388607161896960
- 12m1s https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/test-infra/21044/pull-test-infra-integration/1365397741399707648
- 10m28s https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/test-infra/21044/pull-test-infra-integration/1365401019470581760
- 10m24s https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/test-infra/21044/pull-test-infra-integration/1365405811035082752
- 13m8s https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/test-infra/21044/pull-test-infra-integration/1365409724618510336

**Pull request closed without merging**